alf/algorithms/sac_algorithm.py

```diff
@@ -717,7 +723,10 @@ def _critic_train_step(self, inputs: TimeStep, state: SacCriticState,
             probs = common.expand_dims_as(action_distribution.probs,
                                           target_critics)
             # [B, reward_dim]
-            target_critics = torch.sum(probs * target_critics, dim=1)
+            if target_action_picker is not None:
+                target_critics = target_action_picker(target_critics)
```

Comment:
Together with the comment above (about alpha=0), I think we can just use `probs * target_critics` as long as the `probs` peaks at an entry.